whylog/config/parser_name_manager.py

```diff
@@ -13,19 +15,18 @@ def add_new_parser_name(self, parser_name):
     def is_free_parser_name(self, parser_name, black_list):
         return (parser_name not in self._parsers_name) and (parser_name not in black_list)
 
-    def _try_get_simple_name_else_append_number(self, word, black_list):
-        if self.is_free_parser_name(word, black_list):
-            return word
-        return self._find_free_by_number_appending(word, black_list)
+    def _create_name_from_words(self, words, words_in_name, black_list):
```

Comment:
`words_in_name` looks like a list. Maybe rename to something like `words_count_in_name`?
The same about const WORDS_IN_NAME.